<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KingHelpMain</class>
 <widget class="QWidget" name="KingHelpMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>KingHelpMain</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>链接地址1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图文描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>63</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>产品名字：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>产品编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>器件管理助手</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pronameedit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="procodeEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="prourl1edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>411</width>
     <height>591</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rootIsDecorated">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>21</number>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addbtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatebtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sccutbutton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savebtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="propriceedit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="searchedit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>链接地址2：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prourl2edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>链接地址3：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prourl3edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>链接地址4：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prourl4edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="proimgedit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>9</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pnamechecklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ppricechecklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="purl1checklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="purl2checklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="purl3checklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="purl4checklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgchecklabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>提示：</string>
   </property>
   <widget class="QLabel" name="checkresultlabel">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>11</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
